src/phoenix/server/api/types/Project.py

```diff
@@ -367,5 +369,5 @@ def validate_span_filter_condition(self, condition: str) -> ValidationResult:
             )
 
 
-LLM_TOKEN_COUNT_PROMPT = SpanAttributes.LLM_TOKEN_COUNT_PROMPT
-LLM_TOKEN_COUNT_COMPLETION = SpanAttributes.LLM_TOKEN_COUNT_COMPLETION
+LLM_TOKEN_COUNT_PROMPT = SpanAttributes.LLM_TOKEN_COUNT_PROMPT.split(".")
+LLM_TOKEN_COUNT_COMPLETION = SpanAttributes.LLM_TOKEN_COUNT_COMPLETION.split(".")
```

Comment:
Doesn't this return a list? How does this work if we are using this as the key to a dictionary?

Reply:
yea, this is how JSON extract works in sqlchemy, i.e. by giving it a list